lib/verify/redacted_response.rb

```diff
@@ -0,0 +1,42 @@
+module Verify
+  # Used to report a response from Verify without revealing any Personally
+  # Identifiable Information by replacing any string "values" with the "X"
+  # character. For example:
+  #
+  #   Verify::RedactedResponse.new({"value" => "Secret"}).parameters
+  #   => {"value" => "XXXXXX"}
+  #
+  class RedactedResponse
+    KEYS_TO_REDACT = %w[value postCode].freeze
```

Comment:
It feels like this should be implemented with a whitelist, not a blacklist, in case the response format changes.

Reply:
We could do. I do think the Verify response format is highly unlikely to change, and if it does specs for this class will fail, which should be enough to signal that the redactor will need updating along with the standard `Verify::Response`

Reply:
I'm more worried about them changing on a live system. It's a system we have no control over, and could change in any way, and it would be good not to be leaking PII if that were to happen.